docs/adr/0007-javascript-linting.md

```diff
@@ -0,0 +1,66 @@
+# 7. JavaScript Linting and code style
+
+Date: 2020-08-25
+
+## Status
+
+Accepted
+
+## Context
+
+The issue motivating this decision, and any context that influences or constrains the decision.
```

Comment:
I think you want to remove this line and the one below?

Reply:
Good spot